x/datadeal/client/cli/txRequestDataDeliveryVote.go

```diff
@@ -0,0 +1,44 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdRequestDataDeliveryVote() *cobra.Command {
```

Comment:
The name `RequestDataDeliveryVote` doesn't seem right.
We need to implement a function to retry the data delivery vote.
Umm.. I cannot remember now...
`CmdRetryDataDeliveryVote`?
Let me think about this.

Reply:
Yes, I am also thinking about this function name... If you come up with a good name, let me know.

Reply:
Rename function to `ReRequestDataDeliveryVote`. Please check it :)